tests/tools/oecert/host/host.cpp

```diff
@@ -196,9 +202,41 @@ static void _display_help(const char* cmd)
     printf("\t%s\n", INPUT_PARAM_OPTION_VERBOSE);
 }
 
+// Get full path of oecert running executable, then get enclave filename by:
+// In linux, replace "<path>/oecert" with "<path>/oecert_enc"
```

Comment:
Note that eventually we want to sign oecert_enc. The filename will have "signed" in it.

Reply:
Got it. Can we update the ENCLAVE_FILENAME_SUFFIX to "_enc_signed" in next PR that will sign the enclave file? Will update comment here as well.